site/cli2md/cli2md.go

```diff
@@ -236,13 +236,27 @@ func generateSubcommands(w io.Writer, dir, sectionTitle string, cmds []*kingpin.
 		}
 
 		if first {
-			fmt.Fprintf(w, "\n### %v\n\n", sectionTitle)
+			sectionTitleCutoffIndex := len(sectionTitle)
```

Comment:
is this `strings.TrimSuffix(..., ".")` ?

Reply:
Nice! Thanks.